docs/modules/automation-testing/pages/scripting/add-action-groups-in-appium-script.adoc

```diff
@@ -0,0 +1,144 @@
+= Add action groups in Appium script
+:navtitle: Add action groups in Appium script
+
+To enhance test analysis, you can group related Appium actions. Grouping actions can help in troubleshooting by marking specific phases of a test and aligning them with your testing objectives.
+
+== Action group Appium setting
+
+To add actions within your script, use the below Appium setting to assign actions to a specific group.
+
+[options="header"]
+|=======================
+|Appium setting | Description | Default value
+| `kobiton:stepGroup` | Set to any string value to assign actions to a group with the value as name. | `''` (blank)
+|=======================
+
+== Add action group
+
+Before starting a test action, use the `Update Settings` Appium command to set `'kobiton:stepGroup'` to a string.
+
+.Example (JavaScript)
+[source,javascript]
+
+await driver.updateSettings({'kobiton:setGroup': 'Test case A5423'})
+
+All the update settings command, all actions will be added to the group. To assign subsequent actions to a different group, execute the command again with the new group name.
```

Comment:
Should this say "Using the update settings command, all actions will be added to the group."?